lib/Lift.cpp

```diff
@@ -450,7 +450,7 @@ llvm::Value *StoreNativeValue(llvm::Value *native_val, const ValueDecl &decl,
 
     return mem_ptr;
 
-  // Store it to memory.
+    // Store it to memory.
```

Comment:
Please run the formatter script, found under `./scripts`, not clang-format directly.